src/test/java/teammates/test/cases/action/FeedbackParticipantFeedbackResponseCommentDeleteActionTest.java

```diff
@@ -0,0 +1,263 @@
+        return Const.ActionURIs.FEEDBACK_PARTICIPANT_FEEDBACK_RESPONSE_COMMENT_DELETE;
+    }
+
+    @Override
+    protected void prepareTestData() {
+        super.prepareTestData();
+        dataBundle = loadDataBundle("/FeedbackParticipantFeedbackResponseCommentDeleteTest.json");
+        removeAndRestoreDataBundle(dataBundle);
+    }
+
+    @Override
+    @Test
+    public void testExecuteAndPostProcess() {
+        FeedbackQuestionsDb feedbackQuestionsDb = new FeedbackQuestionsDb();
+        FeedbackResponsesDb feedbackResponsesDb = new FeedbackResponsesDb();
+        FeedbackResponseCommentsDb feedbackResponseCommentsDb = new FeedbackResponseCommentsDb();
+
+        int questionNumber = 1;
+        FeedbackQuestionAttributes feedbackQuestion = feedbackQuestionsDb.getFeedbackQuestion(
+                "First Session", "idOfCourse1", questionNumber);
+
+        String giverEmail = "instructor1@course1.tmt";
+        String receiverEmail = "instructor1@course1.tmt";
+        FeedbackResponseAttributes feedbackResponse = feedbackResponsesDb.getFeedbackResponse(feedbackQuestion.getId(),
+                giverEmail, receiverEmail);
+
+        FeedbackResponseCommentAttributes feedbackResponseComment =
+                dataBundle.feedbackResponseComments.get("comment1FromInstructor1");
+
+        feedbackResponseComment = feedbackResponseCommentsDb.getFeedbackResponseComment(feedbackResponse.getId(),
+                feedbackResponseComment.commentGiver, feedbackResponseComment.createdAt);
+        assertNotNull("response comment not found", feedbackResponseComment);
+
+        InstructorAttributes instructor = dataBundle.instructors.get("instructor1InCourse1");
+        gaeSimulation.loginAsInstructor(instructor.googleId);
+
+        ______TS("Unsuccessful case: not enough parameters");
+
+        verifyAssumptionFailure();
+
+        String[] submissionParams = new String[] {
+                Const.ParamsNames.COURSE_ID, feedbackResponseComment.courseId,
+                Const.ParamsNames.FEEDBACK_SESSION_NAME, feedbackResponseComment.feedbackSessionName,
+                Const.ParamsNames.FEEDBACK_RESPONSE_COMMENT_TEXT, "Comment to first response",
+                Const.ParamsNames.USER_ID, instructor.googleId
+        };
+
+        verifyAssumptionFailure(submissionParams);
+
+        ______TS("Typical successful case when feedback participant is an instructor");
+
+        submissionParams = new String[] {
+                Const.ParamsNames.COURSE_ID, feedbackResponseComment.courseId,
+                Const.ParamsNames.FEEDBACK_SESSION_NAME, feedbackResponseComment.feedbackSessionName,
+                Const.ParamsNames.FEEDBACK_RESPONSE_ID, feedbackResponseComment.feedbackResponseId,
+                Const.ParamsNames.FEEDBACK_RESPONSE_COMMENT_ID, feedbackResponseComment.getId().toString(),
+                Const.ParamsNames.FEEDBACK_RESPONSE_COMMENT_TEXT, feedbackResponseComment.commentText + " (Edited)",
```

Comment:
`(Edited)` is a hard coded magic word, use a const if possible

Reply:
Umm I think it's not really hardcoded magic word because any string can be used since we just want to edit the comment.